prac_07/dynamic_labels.kv

```diff
@@ -0,0 +1,5 @@
+BoxLayout:
+    orientation: 'vertical'
+    BoxLayout:
+        orientation: 'vertical'
+        id: names_box
```

Comment:
Wasn't able to find anything wrong with naming or logic. Also not sure if it matters too much, but noticed a few times that python suggested a newline at the end of the file. Maybe look over these sections and add the newline.